cosmo_tester/test_suites/image_based_tests/aio_create_replace_certs_config_script.py

```diff
@@ -0,0 +1,50 @@
+import argparse
```

Comment:
Can you avoid the duplication for this by merging it with the other one and having a flag (if needed) to control which type of config it outputs?

Reply:
Done.